thirdparty/clamav-db/zimbra-clamav-db/debian/zimbra-clamav-db.postinst

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 if [ "$1" = configure ]; then
-  /bin/chown -R zimbra:zimbra /opt/zimbra/data/clamav
-  /bin/chmod 644 /opt/zimbra/data/clamav/init/*.init
+	/bin/chown -R zimbra:zimbra /opt/zimbra/data/clamav
+	/bin/chmod 644 /opt/zimbra/data/clamav/init/*.init
+	if [ -f /opt/zimbra/data/clamav/db/daily.cvd ]; then
+		rm -rf /opt/zimbra/data/clamav/db/daily.cvd
+	fi
+	if [ -f /opt/zimbra/data/clamav/db/main.cvd ]; then
```

Comment:
we are already doing this as part of package installation, do we need to repeat here?

Reply:
package installation won't install  *.cvd files. It install *.cvd.init files. zmclamdctl script will copy the files(.cvd.init --> .cvd) if they are not present.